crates/burn-fusion/src/client/base.rs

```diff
@@ -14,65 +17,79 @@ pub trait FusionClient: Send + Sync + Clone {
 
     /// Create a new client for the given [fusion device](FusionBackend::FusionDevice).
     fn new(device: <Self::FusionBackend as FusionBackend>::FusionDevice) -> Self;
+
+    fn server(&self) -> &Arc<Mutex<FusionServer<Self::FusionBackend>>>;
+
```

Comment:
This is not a viable option, since the client shouldn't know how the server is actually handled. It might be a channel, an RefCell, etc.

Reply:
Yeah, making this I knew it was kind of hacky... I'll work on a better solution.